routers/user_router.py

```diff
@@ -47,4 +49,16 @@ def delete_user(user_email: str, session: Session = Depends(get_db)) -> user_sch
     except ValueError as e:
         raise HTTPException(status_code=404, detail=str(e))
     return user_schema.UserDeletedResponse(message=f"user with email{user_email}, was successfully deleted")
+
+@router.put("/users/{mentor_email}/{mentee_email}", status_code=status.HTTP_200_OK, response_model=user_schema.MentorAssignedResponse)
```

Comment:
1- this endpoint is very complicated in my opinion. 
2- I do not even think we need an endpoint for this.  

I believe you just need to do a PUT update from https://github.com/PkFokam-Alumni-Network/CommunityBackendService/issues/5 unless you see a use case for this endpoint 
cc @bjoelfra @atakoutene